environments/neutron-ml2-vpp.yaml

```diff
@@ -0,0 +1,22 @@
+# Environment file used to enable networking-vpp ML2 mechanism driver
+
+resource_registry:
+  OS::TripleO::Services::NeutronOvsAgent: OS::Heat::None
+  OS::TripleO::Services::ComputeNeutronOvsAgent: OS::Heat::None
+  OS::TripleO::Services::NeutronVppAgent: ../puppet/services/neutron-vpp-agent.yaml
+  OS::TripleO::Services::Etcd: ../puppet/services/etcd.yaml
+  OS::TripleO::Services::Vpp: ../puppet/services/vpp.yaml
+
+parameter_defaults:
+  #Comma delimited list of <physical_network>:<VPP Interface>.
+  #Example: "datacentre:GigabitEthernet2/2/0"
+  #NeutronVPPAgentPhysnets: ""
+
+  NeutronMechanismDrivers: vpp
+  NeutronNetworkType: vlan
+  NeutronServicePlugins: router
+  NeutronTypeDrivers: vlan,flat
+  ExtraConfig:
+    # Use Linux Bridge driver for DHCP and L3 agent.
+    neutron::agents::dhcp::interface_driver: "neutron.agent.linux.interface.BridgeInterfaceDriver"
```

Comment:
Double check this will acutally work because with ODL FDIO L2 we always hardcode OVS driver when the namespace comes up for external routing interface.